Resources/Prototypes/Entities/Constructible/Walls/signs.yml

```diff
@@ -39,6 +32,12 @@
     shapes:
     - !type:PhysShapeAabb
       bounds: "-0.1,-0.4,0.1,0.4"
+      layer:
```

Comment:
Why not use yaml reference ? https://en.wikipedia.org/wiki/YAML#Advanced_components